woocommerce/woocommerce-gutenberg-products-block
# Update changelog, bump version to 1.4.0-rc

Update the version number, changelog, and translations to prepare to release 1.4.0

**To test**

- [View the changelog section in the readme.txt,](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/update/changelog-1.4.0-rc/readme.txt) make sure the entry for 1.4.0 makes sense
- Install this branch, check that the version in wp-admin says 1.4.0
- Check at least one translation pack by switching your site to one of the languages below, add some blocks and make sure they're all translated. Note: block names are translated too, so if you're used to typing `/product` to get the list of blocks, that might not work.
